scheduler/metrics/metrics.go

```diff
@@ -30,6 +30,14 @@ import (
 	"d7y.io/dragonfly/v2/version"
 )
 
+const (
+	// TypeBackToSource is metrics type of back-to-source.
+	TypeBackToSource = "BACK_TO_SOURCE"
+
+	// TypeP2p is metrics type of not back-to-source.
+	TypeP2p = "P2P"
```

Comment:
Remove type definition and use `commonv2.TrafficType_BACK_TO_SOURCE` and `commonv2.TrafficType_LOCAL_PEER`.